lib/dal/src/schema/variant.rs

```diff
@@ -382,6 +419,26 @@ impl SchemaVariant {
         &self.category
     }
 
+    pub fn timestamp(&self) -> Timestamp {
+        self.timestamp
+    }
+
+    pub fn display_name(&self) -> Option<String> {
+        self.display_name.clone()
+    }
+
+    pub fn link(&self) -> Option<String> {
+        self.link.clone()
+    }
+
+    pub fn description(&self) -> Option<String> {
+        self.description.clone()
+    }
```

Comment:
nit: feel free to ignore. We could use `self.<option-field>.as_deref()` so that we allow the caller to decide to clone.